pennylane_lightning/lightning_gpu/_measurements.py

```diff
@@ -15,8 +15,24 @@
 Class implementation for state vector measurements.
 """
 
+try:
+    from pennylane_lightning.lightning_gpu_ops import MeasurementsC64, MeasurementsC128
+
+    try:
+        from pennylane_lightning.lightning_gpu_ops import MeasurementsMPIC64, MeasurementsMPIC128
+
+        MPI_SUPPORT = True
+    except ImportError:
```

Comment:
Could we update this line with current implementations [here](https://github.com/PennyLaneAI/pennylane-lightning/blob/3d005f05d25aef866e1cd5418093283f937d28bc/pennylane_lightning/lightning_gpu/lightning_gpu.py#L77-L80)?

Reply:
Done!